backend.native/build.gradle

```diff
@@ -156,6 +156,33 @@ task unzipStdlibSources(type: Copy) {
     destinationDir commonSrc
 }
 
+final List<File> stdLibSrc = [
```

Comment:
I don't think it's a good idea to always print `"##teamcity..."`, specially if you build it locally

Reply:
The task or print could be surrounded by the check for env or property existence "TEAMCITY_BUILD_PROPERTIES_FILE"

Reply:
Added the `if`.